### ISSUE TITLE
The DisableSD2IECCommands option makes the Pi1541 hang

### ISSUE THREAD

**Issue by perinoid** [pi1541/Pi1541#95]
I tried to run my Pi1541 with DisableSD2IECCommands=1. It doesn't work. When it boots, it scans the i2c bus to find the OLED display (with success), then it loads the kernal, goes to filefrowser and hangs. Neither on-PCB buttons nor keyboard work. The device is also nor visible on the IEC bus.

**Comment by perinoid** [comment]
One more thing - tried on both 1.13 and 1.14 - the same.

**Comment by pi1541** [comment]
Fixed in v1.15

**Comment by perinoid** [comment]
Nope. It isn't. My Pi1541 behaves in the same way with 1.15 like with 1.14 and 1.13 - it freezes immediately after it's started :-(
